deploy/007_deploy_clay_staking_rewards.js

```diff
@@ -13,12 +13,13 @@ const func = async function (hre) {
 
     const ClayToken = await deployments.get("ClayToken");
 
-    let sumeroLpToken = { address: "0x8Ad9995145D24bA99326ea637218e1fc86C6b1fb" };
+    let sumeroLpToken = { address: "0x8556Fa1107401dAD95D67B1a9bbF49d87Fa18b4c" };
 
     // change days depending on when you want staking to end
-    const days = 16;
-    const currentTimestamp = Date.now() / 1000;
-    const expirationTimestamp = Math.floor(currentTimestamp + (days * 24 * 3600));
```

Comment:
I think we can revert the changes here.

Reply:
I can remove the comments, we decided to use a timestamp instead of days, to get an accurate expiry time.